src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -272,14 +274,6 @@ void Tailsitter::update_transition_state()
 		_mc_roll_weight = 1.0f;
 		_mc_pitch_weight = 1.0f;
 
-		// check front transition timeout
```

Comment:
There's no timeout part when I revised the code... 
Do I need revise it again base on the current code and then pull request?

this is my first time to pull request...not familiar it at all... sorry for causing any trouble.